version.go

```diff
@@ -0,0 +1,4 @@
+package gocql
+
+const DRIVER_NAME = "scylladb-gocql"
+const DRIVER_VERSION = "1.7.3"
```

Comment:
nit: we'll need to remember to update it every release.

Reply:
Yes, this is the solution that we've agreed upon with @avelanarius.